components/standings/commons/standings_parse_wiki.lua

```diff
@@ -176,4 +179,22 @@ function StandingsParseWiki.makeScoringFunction(tabletype, args)
 	error('Unknown table type')
 end
 
+---@param args table
+---@param tableType StandingsTableTypes
+---@return StandingsTiebreaker[]
+function StandingsParseWiki.parseTiebreakers(args, tableType)
+	local tiebreakers = {}
+	for _, tiebreaker in ipairs(args.tiebreakers or {}) do
```

Comment:
how can args.tiebreakers be a table?
wouldn't we need to parse a json/csv or something like that here?